Capstone Module 1

BregornOriginal opened the pull request:
# First Capstone for Microverse Bootcamp!🥳

## This Capstone was made in 3 days. The next features are created.

✅ 2 pages were created for Mobile (up to 768px wide) and Desktop (768px or wider) versions:

1. Home page
2. About page

✅ The home page has a link in the menu to the about page.
✅ The logo of the page (ring) links to the home page

- Mobile menu
✅  The hamburger button when is clicked shows the menu over the page
✅  The mobile menu has a close button, that closes the menu

- Dynamic page
✅ The section "Featured Actors" was created dynamically in JavaScript
✅ An object in JavaScript was created to store the information of the cars

**TECHNOLOGIES USED**
🔷HTML
🔷CSS
🔷JAVASCRIPT

**TOOLS USED**
💠VISUAL STUDIO CODE
💠GITHUB-DESKTOP
💠GITHUB-ACTIONS
💠LINTERS
💠LOOM

### This is the link to the [Loom video](https://www.loom.com/share/a2bce50c1ee84fbcb5723f6de01176ed), hope you enjoy it!

BregornOriginal committed:
Section "Featured Actors" with dynamic HTML

- The MORE button of Featured Actors can show more or less actors when it's pressed.

BregornOriginal committed:
added photos to personalize the topic

- Added effect hover to the Ring icon links
- Added hover effect to the featured cards
- Fixed some minnor bugs

BregornOriginal committed:
Linter errors corrected

Ismailco reviewed (commented):
Hi @BregornOriginal,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that dos do not use Github flow, please consider opening a PR from your branch to the ```main``` branch

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

BregornOriginal commented:
Thank you @Ismailco for the review, I have corrected the GitHub Flow

ShadyShawkat commented on index.html:
```diff
@@ -134,6 +134,60 @@ <h5>SEE THE WHOLE PROGRAM</h5>
 				<a class="join-button" href=""> Join LOTR Celebration 2022</a>
 			</section>
 		</main>
+		<section id="featured-actors" class="featured-actors">
+			<h2>Featured Characters</h2>
+			<hr />
```
- [ ] Kindly add the top navigation bar which contains the social media icons, to match the design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/169328800-c6b6580c-f4ea-42ff-99b2-b6edb802ff80.png) | ![image](https://user-images.githubusercontent.com/62290545/169328689-036413fb-cfc5-4641-a0f3-4a3983a8884a.png) |

ShadyShawkat commented on index.html:
```diff
@@ -134,6 +134,60 @@ <h5>SEE THE WHOLE PROGRAM</h5>
+		<section id="featured-actors" class="featured-actors">
+			<h2>Featured Characters</h2>
+			<hr />
+		</section>
+		<div class="see-more">
+			<button id="see-more-button" type="button">MORE</button>
+		</div>
```
- [ ] Please make sure that the footer matches the design on the "about" page.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/169329294-8b0927ff-df26-459d-96a6-39ef23667243.png) | ![image](https://user-images.githubusercontent.com/62290545/169329082-f666dff0-7fa1-40c1-af87-298a4f77907b.png) |

ShadyShawkat commented on index.html:
```diff
@@ -134,6 +134,60 @@ <h5>SEE THE WHOLE PROGRAM</h5>
+		<footer>
+			<section class="partners">
+				<div class="mobile-container">
+					<h3>Partner</h3>
+					<hr />
+					<div class="footer-logos">
```
- [ ] Please make sure that the cards displayed in the about page appear side by side on desktop view to match the design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/169329913-4c2509c6-e15a-48e3-8003-9928d3250d74.png) | ![image](https://user-images.githubusercontent.com/62290545/169329666-c4fd13ba-bc25-4c68-b484-14fa4ae61ffc.png) |

ShadyShawkat reviewed (changes requested):
## Hi @BregornOriginal, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

![Image](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## Required Changes ♻️
- [x] Please make sure to stick to [this readme](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse, Kindly make sure to fill each section of the readme with the appropriate data to suit your project.
- [x] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BregornOriginal committed:
Added nav-bar whit social icons

BregornOriginal committed:
semanthic tags in html corrected

BregornOriginal committed:
Added footer bar for about section in mobile

- The two images in the about section of desktop version now are side by side

BregornOriginal committed:
linters corrected

BregornOriginal committed:
corrected path of icon in the index page

BregornOriginal committed:
fix bug

Wes-Isaac commented on html/about.html:
```diff
@@ -101,13 +118,15 @@ <h4>
+					</div>
 				</div>
 			</article>
 		</section>
```
- [ ] As the previous reviewer pointed out please fix the footer of the about page according to the design.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/169388674-b88eef14-3393-4fec-9b89-99cdf89b5e63.png)

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/169388786-c11738e3-77a4-45d1-a402-ba977b8d5200.png)

Wes-Isaac commented on index.html:
```diff
@@ -134,6 +151,60 @@ <h5>SEE THE WHOLE PROGRAM</h5>
 				<a class="join-button" href=""> Join LOTR Celebration 2022</a>
 			</section>
 		</main>
+		<section id="featured-actors" class="featured-actors">
+			<h2>Featured Characters</h2>
+			<hr />
```
- [ ] [_OPTIONAL_] It's better to use two columns for the Featured Characters part and also better to include the background checker images for the images of the characters.

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/169389587-815d8ca7-0386-4f2d-b89a-e6aaa00d0e4e.png)

### Your Implementation 
![image](https://user-images.githubusercontent.com/59416136/169389702-685ad54e-89b8-4e7d-b2d4-c87e9cdd4532.png)

BregornOriginal replied:
hey @Wes-Isaac ! Thank you for the review! that idea about the columns was great, it's more beautiful now. Hope now is all finished!
Hi @ShadyShawkat ! I was confused whit the footer because the different versions and pages have different styles. So I upload it in the wrong way. Thank you for noticing that!

Wes-Isaac reviewed (changes requested):
Hi @BregornOriginal, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes

- [ ] Please make sure to stick to [this readme](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse, Kindly make sure to fill each section of the readme with the appropriate data.

_Check the comments under the review._


You can also consider:

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

BregornOriginal committed:
Added footer NOW CORRECTLY for both versions

BregornOriginal committed:
Optional changes

- Now the grid in Featured Characters is showed in 2 columns, in this way has a better view!
- The images now has a background checker images for the images of the characters.

BregornOriginal committed:
Linters check

BregornOriginal committed:
Linters corrected

Lucas-Masaba reviewed (changes requested):
# Hello @BregornOriginal  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![almost-there](https://media.giphy.com/media/dEPGknRiEniec/giphy.gif)

 ## Highlights 🍾 
- Implemented most of the changes requested by the previous reviewer 👍 
- Linter checks passing ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] As mentioned earlier by the past two reviewers, you have to add a README.md file in this branch `featured-actors` that follows the microverse format as seen in this [link](https://github.com/microverseinc/readme-template/blob/master/README.md). If this is not very clear, please reach out to me or anyone else. This is all that is left for your project to be marked as complete 🙏 .

BregornOriginal committed:
Update README

BregornOriginal commented:
Hi! @Lucas-Masaba How are you?
I was updating my README file in the main branch, but that was wrong. So now, I have copied that content in this branch ``featured-actors``. Please, can you tell me if now it's ok? Because I am going to use an extra request review for this.
Thank you!
Best,
Julio

Lucas-Masaba commented:
Hello @BregornOriginal, sorry for the late reply, I got caught up. I have looked at your README and I believe it is ready for a re-review.

BregornOriginal commented:
No problem. Ok, @Lucas-Masaba! Thank you for the review and your time!

nicupop729 reviewed (approved):
Hi @BregornOriginal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/5hgYDDh5oqbmE4OKJ3/giphy.gif"/>

## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lucas-Masaba commented:
No problem @BregornOriginal , now I believe you can take a break 🏄
